Dockerfiles/agent/OPENSHIFT.md

```diff
@@ -0,0 +1,54 @@
+# Openshift installation and configuration instructions
+
+Starting with version 6.1, the Datadog Agent supports monitoring OpenShift Origin and Enterprise clusters. Depending on your needs and the [security constraints](https://docs.openshift.org/latest/admin_guide/manage_scc.html) of your cluster, we support three deployment scenarios:
+
+| Security Context Constraints   | Restricted | Host network | Custom |
+|--------------------------------|:----------:|:------------:|:------:|
+| Kubernetes layer monitoring    | ✅         | ✅          | ✅     |
+| Kubernetes-based Autodiscovery | ✅         | ✅          | ✅     |
+| Dogstatsd intake               | 🔶         | ✅          | ✅     |
+| APM trace intake               | 🔶         | ✅          | ✅     |
+| Logs network intake            | 🔶         | ✅          | ✅     |
+| Host network metrics           | ❌         | ❌          | ✅     |
+| Docker layer monitoring        | ❌         | ❌          | ✅     |
+| Container logs collection      | ❌         | ❌          | ✅     |
+| Live Container monitoring      | ❌         | ❌          | ✅     |
+| Live Process monitoring        | ❌         | ❌          | ✅     |
+
+## General information
+
+- You should first refer to the [common installation instructions](README.md), and its [Kubernetes section](README.md#Kubernetes)
+- We only support full operations on OpenShift 3.7.0 and later, as we rely on new monitoring endpoints introduced in this version
+- On OpenShift 3.7, you will need to change the provided RBAC files to refer to the `rbac.authorization.k8s.io/v1beta` apiVersion instead of `rbac.authorization.k8s.io/v1`
```

Comment:
Should we switch them to `rbac.authorization.k8s.io/v1beta` to avoid this? (we're still using it for the daemonset)

Reply:
IIRC the apply will fail on k8s 1.8+ if we do so, as it does not rewrite the new apiversion :/

Reply:
maybe we should pipe it through `kubectl convert`

Reply:
Great idea, `kubectl convert -f clusterrolebinding.yaml` indeed injects `apiVersion: rbac.authorization.k8s.io/v1beta1` when connected to a 1.7 cluster.

Unfortunately, the stable `oc` client is built on top of `kubectl` 1.7.6 and does not handle the new apiversion: 

> error: unable to decode "clusterrolebinding.yaml": no kind "ClusterRoleBinding" is registered for version "rbac.authorization.k8s.io/v1"

```
oc v3.7.1+ab0f056
kubernetes v1.7.6+a08f5eeb62
features: Basic-Auth

Server https://192.168.33.22:8443
openshift v3.7.1+a8deba5-34
kubernetes v1.7.6+a08f5eeb62
```

We could provide a copy-pasta piping them into `sed` though